README.markdown

```diff
@@ -7977,6 +7977,9 @@ Similar to the [ngx.timer.at](#ngxtimerat) API function, but
 1. `delay` *cannot* be zero,
 1. timer will be created every `delay` seconds until the current Nginx worker process starts exiting.
 
+Like [ngx.timer.at](#ngxtimerat), the `callback` argument will be called
+automatically with the arguments `premature`, `user_arg1`, `user_arg2`, etc.
```

Comment:
Please do not edit this file directly. Instead, make your edits to the [HttpLuaModule.wiki](https://github.com/openresty/lua-nginx-module/blob/master/doc/HttpLuaModule.wiki) file, and generate the README file with the [wiki2markdown.pl](https://github.com/openresty/openresty-devel-utils/blob/master/wiki2markdown.pl) utility.